templates/includes/company/page_header.tx

```diff
@@ -31,6 +31,16 @@
                     % }
                 % }
             % }
+            % if $c.config.feature.enable_oe_auth_code {
+                % if $company.type == "registered-overseas-entity" {
+                      <a href="<% $c.url_for.to_abs %>" id="request_auth_code_roe" class="govuk-button"
+                          % if $c.config.piwik.embed {
+                              onclick="javascript:_paq.push(['trackEvent', 'RequestAuthcodeROE']);"
+                          % }
+                      ><% l('Request an authentication code') %> 
```

Comment:
Guessing this means that button text is 'Request an authentication code'?

This matches AC1 of the story but then the screenshot suggests button text is just 'Request authentication code'.

Please can you clarify which is correct and request analyst to update the ticket. Thanks.

Reply:
AC1 has been updated to confirm "Request authentication code" as the correct text.
PR updated